visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -2124,6 +2186,7 @@ namespace pcl
           * \param[in] actor a pointer to the vtk actor object
           * \param[in] viewport the view port where the actor should be added to (default: all)
           */
+		PCL_DEPRECATED(1, 14, "deprecated");
```

Comment:
Add nicer message to the user (there should be none, but still). Compiler already says "using deprecated function".